target/linux/mediatek/image/filogic.mk

```diff
@@ -1311,6 +1311,27 @@ define Device/ruijie_rg-x60-pro
 endef
 TARGET_DEVICES += ruijie_rg-x60-pro
 
+define Device/tenbay_wr3000k
+  DEVICE_VENDOR := Tenbay
+  DEVICE_MODEL := WR3000K
+  DEVICE_DTS := mt7981b-tenbay-wr3000k
+  DEVICE_DTS_DIR := ../dts
+  DEVICE_PACKAGES := kmod-mt7915e kmod-mt7981-firmware mt7981-wo-firmware
+  UBINIZE_OPTS := -E 5
+  BLOCKSIZE := 128k
+  PAGESIZE := 2048
+  IMAGE_SIZE := 49152k
+  KERNEL_IN_UBI := 1
+  IMAGES += factory.bin
```

Comment:
Where is the factory.bin used?

Reply:
it used for uboot flash ubi(s) partitoin

Reply:
it used for uboot flash ubi(s) partition

Reply:
Is sysupgrade.bin enough?

Reply:
factory.bin can use `dd` to flash mtd directly, so i think we can reserve it.